src/open_inwoner/accounts/tests/test_oidc_views.py

```diff
@@ -159,13 +295,67 @@ def test_new_user_is_created_when_new_email(
         )
 
         self.assertRedirects(
-            callback_response, reverse("pages-root"), fetch_redirect_response=False
+            callback_response, reverse("admin:index"), fetch_redirect_response=True
+        )
+
+        new_user = User.objects.filter(email="new_user@example.com").first()
```

Comment:
Here is is more clear as `new_user` and the test title work together. 

Nevertheless the filter()/first() is a bit floppy and should just be a .get() (note there is even a `assertIsNotNone(new_user)` there to work around .first()  :frowning_face: